carrot_cli/tests/unit/test/test_test_command.py

```diff
@@ -1143,6 +1144,68 @@ def test_run(run_data, caplog):
                 sort_keys=True,
             ),
         },
+        {
+            "args": [
+                "test",
+                "find_runs",
+                "cd987859-06fe-4b1a-9e96-47d4f36bf819",
+                "--name",
+                "Queen of Bright Moon run",
+                "--status",
+                "succeeded",
+                "--test_input",
+                "tests/data/mock_test_input.json",
+                "--test_options",
+                "tests/data/mock_test_options.json",
+                "--eval_input",
+                "tests/data/mock_eval_input.json",
+                "--eval_options",
+                "tests/data/mock_eval_options.json",
+                "--test_cromwell_job_id",
+                "d9855002-6b71-429c-a4de-8e90222488cd",
+                "--eval_cromwell_job_id",
+                "03958293-6b71-429c-a4de-8e90222488cd",
+                "--created_before",
+                "2020-10-00T00:00:00.000000",
+                "--created_after",
+                "2020-09-00T00:00:00.000000",
+                "--created_by",
+                "glimmer@example.com",
+                "--finished_before",
+                "2020-10-00T00:00:00.000000",
+                "--finished_after",
+                "2020-09-00T00:00:00.000000",
+                "--sort",
+                "asc(name)",
+                "--limit",
+                1,
+                "--offset",
+                0,
+                "--zip_csv",
+                "csvs.zip"
+            ],
+            "params": [
+                "cd987859-06fe-4b1a-9e96-47d4f36bf819",
+                "Queen of Bright Moon run",
+                "succeeded",
+                {"in_greeted": "Cool Person"},
+                {"option": "other_value"},
+                {"in_output_filename": "test_greeting.txt"},
+                {"option": "value"},
+                "d9855002-6b71-429c-a4de-8e90222488cd",
+                "03958293-6b71-429c-a4de-8e90222488cd",
+                "2020-10-00T00:00:00.000000",
+                "2020-09-00T00:00:00.000000",
+                "glimmer@example.com",
+                "2020-10-00T00:00:00.000000",
+                "2020-09-00T00:00:00.000000",
+                "asc(name)",
+                1,
+                0,
+                "csvs.zip",
```

Comment:
The unit tests in the CLI have a ton of mocking in them partly to make sure they're testing specifically the functionality of the function that's being tested and partly to avoid having to spin up a carrot server to run the tests.  In this particular test, it's basically just checking, "When this function receives a parameter for zip-csv, does it call the function it's supposed to and return the data it's supposed to."

Reply:
Admittedly, I'm not totally convinced the approach I've taken with a lot of these tests is the ideal one.  I just haven't taken the time to evaluate that properly and figure out what a better approach would be.